fuel-vm/src/tests/blockchain.rs

```diff
@@ -1322,8 +1322,8 @@ fn smo_instruction_works() {
         #[rustfmt::skip]
         let script = vec![
```

Comment:
Could you also fix the "TODO" below, please?=) It requires checking of the data output=)

Reply:
Done. Also fixed some issues I found after that.